### DISCUSSION TITLE
Problem with theme

### DISCUSSION THREAD

**Discussion by williamuy** [shadcn-ui/ui#4065] [Q&A]
Hello Everyone.

For some reason, when i copy a component, the color scheme is not the same. 

I have setup the theme provider and global css correctly, but when i copy the card component from the website, it is different.

![image](https://github.com/shadcn-ui/ui/assets/131928949/3f4c6e96-f84c-4187-8271-d526a0f232a7)

![image](https://github.com/shadcn-ui/ui/assets/131928949/e612e0d1-868d-49de-adc7-db47e71b65b8)

Any help is greatly appreciateed

**Comment by ezeparziale** [comment]
It seems that the error is in your globals.css

**Reply by williamuy** [reply]
it was actually the component itself. I did the quick install firstly, and got that. but when i did the manual version it worked fine. 

But thanks!
